recipes/runtimeqml/all/conandata.yml

```diff
@@ -0,0 +1,10 @@
+sources:
+  "latest":
+    url: https://github.com/GIPdA/runtimeqml/archive/refs/heads/master.zip
+    sha256: "1bf2b8a6e9b9114ec94208db63a0f8de8becc2d85ca57661099d81e3122ecc47"
```

Comment:
```suggestion
```
Prefer only official versions: https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-version-should-packages-use-for-libraries-without-official-releases